packages/syntax-core/src/LinkButton/LinkButton.tsx

```diff
@@ -45,6 +46,19 @@ export default function LinkButton({
    *
    */
   target?: "_blank" | "_self" | "_parent" | "_top";
+  /**
+   * The target attribute specifies where to open the linked document.
+   *
+   */
+  rel?:
```

Comment:
@somethiiing this should be changed to `rel?: HTMLAttributes<HTMLAnchorElement>["rel"]` which will resolve to `rel?: string` since `rel` can take multiple values:
https://www.elegantthemes.com/blog/wordpress/rel-noopener-noreferrer-nofollow